src/plugins/drawing/model/drawing-content.ts

```diff
@@ -254,11 +289,27 @@ export const DrawingContentModel = TileContentModel
           const newIds: string[] = [];
           forEachObjectId(ids, (object) => {
             if (object) {
+              let newObject: DrawingObjectType;
               const snap = getSnapshot(object);
```

Comment:
Super minor, but this could be moved into the `else` below.

Reply:
Both the if and else create a `newObject` was the idea here.